packages/embark-compiler/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "embark-compiler",
+  "version": "4.0.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark is a framework that allows you to easily develop and deploy DApps",
+  "homepage": "https://embark.status.im/",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "license": "MIT",
+  "repository": "github:embark-framework/embark",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
+    "ci": "npm run qa",
+    "clean": "npx rimraf .nyc_output coverage dist embark-*.tgz package",
```

Since it won't have its own test suite:
```suggestion
    "clean": "npx rimraf dist embark-*.tgz package",
```